english/run_out_the_door/run_out_the_door.md

```diff
@@ -1,7 +1,9 @@
 You immediately jump up and run out the nearest exit!
 
-You burst out the door and the entire world is in front of you.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.